Xamarin.Forms.Core/TappedEventArgs.cs

```diff
@@ -4,11 +4,13 @@ namespace Xamarin.Forms
 {
 	public class TappedEventArgs : EventArgs
 	{
-		public TappedEventArgs(object parameter)
+		public TappedEventArgs(object parameter, Point position)
```

Comment:
Is there any advantage to have two seperate constructors instead of using default value parameters? 
`public TappedEventArgs(object parameter, Point position = new Point())`

Reply:
Most likely the default value is sufficient.
Thanks.